.travis.yml

```diff
@@ -7,7 +7,7 @@ go:
 
 matrix:
   include:
-    - go: 1.13.x
+    - go: 1.x
```

Comment:
If go 1.15 comes out which may introduce changes to go modules, then this might silently break CI via `make verify-dependencies`.

Reply:
good point, fixed